proto/internal/core/v1/core.proto

```diff
@@ -12,6 +12,8 @@ message RelationTuple {
 
   /** subject is the subject for the tuple */
   ObjectAndRelation subject = 2 [ (validate.rules).message.required = true ];
+
+  bytes caveat = 3 [(validate.rules).bytes = {min_len: 0, max_len: 4096}];
```

Comment:
Add a comment

Reply:
addressed in https://github.com/authzed/spicedb/pull/773/commits/16db8c396bfac1613471584d50b7bb3d28cd456b